env0/utils.go

```diff
@@ -242,6 +242,12 @@ func writeResourceData(i interface{}, d *schema.ResourceData) error {
 
 		if fieldType.Kind() == reflect.Ptr {
 			if field.IsNil() {
+				if !parsedField.omitEmpty {
```

Comment:
Unless omitempty is set. (In the PR notice I added some additional omitempty to some of the struct).
If the value is nil, it makes sense to set the field value to nil. This will actually set it to the default value as defined in the schema.